src/pages/OwnerPage/Header/HeaderBanners/ReachingUploadLimit/ReachingUploadLimit.jsx

```diff
@@ -1,19 +1,21 @@
+import PropTypes from 'prop-types'
+
 import A from 'ui/A'
 import Banner from 'ui/Banner'
 import BannerContent from 'ui/Banner/BannerContent'
 import BannerHeading from 'ui/Banner/BannerHeading'
 import Button from 'ui/Button'
 
-const ExceededUploadsAlert = () => (
+const ExceededUploadsAlert = ({ planName, monthlyUploadLimit }) => (
```

Comment:
Also this has a different component name than the file, I believe we want to call this component `ReachingUploadLimit` and probably missed it, could you please change it?